hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -88,9 +88,20 @@
 
     KAFKA_AUTHORIZATION_ENABLED("kafka.authorization.enabled", false),
     KAFKA_AUTHORIZATION_MECHANISM("kafka.authorization.mechanism", "PLAIN"),
+    //SASL AUTHENTICATION
     KAFKA_AUTHORIZATION_PROTOCOL("kafka.authorization.protocol", "SASL_PLAINTEXT"),
     KAFKA_AUTHORIZATION_USERNAME("kafka.authorization.username", "username"),
     KAFKA_AUTHORIZATION_PASSWORD("kafka.authorization.password", "password"),
+    //SSL AUTHENTICATION
+    //KAFKA_AUTHORIZATION_PROTOCOL("kafka.authorization.mechanism", "SSL"),
```

Comment:
Is there any reason to leave this line commented out? Btw, the enum name is inconsistent with the property name.

Reply:
you don't have to use this line at all, the property is read from the management.yaml file.